python/pyspark/mllib/linalg/__init__.py

```diff
@@ -309,17 +354,18 @@ class DenseVector(Vector):
     DenseVector([-1.0, -2.0])
     """
 
-    def __init__(self, ar):
+    def __init__(self, ar: Union[bytes, np.ndarray, Iterable[float]]):
```

Comment:
This is intentionally simplified compared to `pyi` implementation ‒ there seems to be no advantage of keeping original overloads.